bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/api2/ShellyBluApi.java

```diff
@@ -165,7 +165,7 @@ public ShellyDeviceProfile getDeviceProfile(String thingType, @Nullable ShellySe
         }
 
         if (!connected) {
-            throw new ShellyApiException("BLU Device not yet connected");
+            // throw new ShellyApiException("BLU Device not yet connected");
```

Comment:
Can you remove this line as well as the lines above and below as it's now an empty block?

Reply:
fixed